pupil_src/shared_modules/tasklib/manager.py

```diff
@@ -122,6 +125,10 @@ def add_task(self, task_new, identifier: str):
         UniqueTaskManager._patch_task(task_new, identifier)
         task_duplicated = self._get_duplicated_task(identifier)
         if task_duplicated is not None:
+            logger.debug(
+                f"{self} replacing queued task '{task_duplicated.process.name}' "
+                f"({identifier})"
+            )
```

Comment:
The `TaskInterface` does not define `.process`, which is only valid for background tasks.
Also I would add info if the task was already running vs still queued.
I wouldn't add `{self}` as we do that nowhere, but maybe we can actually add the line-number to the general logging format in the future.

```suggestion
            state = "running" if task_duplicated.running else "queued"
            logger.debug(f"Replacing {state} task with ID '{identifier}'")
```
Example output when calculating a calibration twice:
```
player - [DEBUG] tasklib.manager: Replacing running task with ID 'bd1f4749-5f55-484e-8131-6f8c87b8a5dd'
```